### PR TITLE
[treadmill] Add number of calibration/warm-up requests as parameters

### PR DESCRIPTION
**Pull request by yqzhang**
- Add number of calibration/warm-up requests are parameters
- This shouldn't change the original behavior of the program as the default values are kept the same

### CODE REVIEW

File: RandomEngine.cpp [hunk 1]
```diff
@@ -14,7 +14,9 @@
 
 #include <folly/Likely.h>
 
-DEFINE_uint64(random_seed, ULLONG_MAX, "seed for random engines");
+DEFINE_uint64(random_seed,
```
Comment (lpkruger) [inline RandomEngine.cpp]: Please stop making random unrelated formatting changes.

Comment (yqzhang) [inline RandomEngine.cpp]: Will do.

**Comment by yqzhang** [comment]
I'm still trying to fix the build due to the recent change in fbthrift.

https://github.com/facebook/fbthrift/commit/3ee61f62fe9f6b36f0f882efe838ee6f9ea8159a

**Comment by yqzhang** [comment]
- Reverted the unrelated formatting change as @lpkruger suggested
- Fixed the build

Ready for review
